doc/nrf/gs_ins_windows.rst

```diff
@@ -46,113 +52,190 @@ To set up the toolchain, complete the following steps:
-   To do so, open a command prompt and enter the following commands (assuming that you have installed the toolchain to |tcfolder|; if not, change the value for GNUARMEMB_TOOLCHAIN_PATH)::
+   To do so, open a |bash| and enter the following commands (assuming that you have installed the toolchain to |tcfolder_cc|; if not, change the value for GNUARMEMB_TOOLCHAIN_PATH):
 
-     set ZEPHYR_TOOLCHAIN_VARIANT=gnuarmemb
-     set GNUARMEMB_TOOLCHAIN_PATH=c:\gnuarmemb
+   .. parsed-literal::
+      :class: highlight
 
-#. Instead of setting the environment variables every time you open a command prompt, add them as system environment variables or define them in the ``zephyrrc.cmd`` file as described in `Setting up the build environment`_.
+       set ZEPHYR_TOOLCHAIN_VARIANT=gnuarmemb
+       set GNUARMEMB_TOOLCHAIN_PATH=\ |tcfolder|
 
-.. _cloning_the_repositories_win:
+#. Instead of setting the environment variables every time you open a |bash|, |system_vars| define them in the |rcfile| file as described in `Setting up the build environment`_.
 
-.. |bash| replace:: GIT bash
+.. toolchain_end
+
+.. _cloning_the_repositories_win:
 
 .. cloning_start
 
-Cloning the repositories
-************************
+Getting the |NCS| code
+**********************
 
 The |NCS| consists of the following repositories:
 
-* |ncs_zephyr_repo|
 * |ncs_repo|
+* |ncs_zephyr_repo|
 * |ncs_mcuboot_repo|
 * |ncs_nrfxlib_repo|
 
-.. note::
-   The following instructions explain how to clone the repositories to be able to develop your own applications based on the supplied libraries and samples.
-   If you want to actively contribute to the |NCS| development and submit the changes that you make, skip this step and :ref:`fork and clone the repositories <gs_installing_forking>` instead.
+Every |NCS| release consists of a combination of these repositories at different revisions.
+
+The latest state of development is on the master branches of the repositories.
+Note that this might not be a stable state.
+To ensure a usable state, the fw-nrfconnect-nrf defines the compatible states of the other repositories.
+However, this state is not necessarily tested.
+Therefore, unless you are familiar with the development process, you should always work with a specific release of the |NCS|.
+
+To manage the combination of repositories and versions, the |NCS| uses :ref:`zephyr:west`.
+The main repository, fw-nrfconnect-nrf, contains a `West manifest file`_ that defines the versions of all repositories.
+This means that fw-nrfconnect-nrf acts as manifest repository in West (see :ref:`Repository structure <zephyr:west-struct>`), while the other repositories are project repositories.
+To work with a specific release, you initialize West with the corresponding version of the manifest file, and it will check out the correct versions of the project repositories for you.
```

Comment:
I think we should rephrase this because typically users will use `west init` only once and then hop across releases and master directly by using Git in the `nrf` repository:

"To work with a specific release, you check the corresponding tag in the nrf repository and then use west to bring all projects to the corresponding revision specified in the manifest file inside the nrf repository"

Reply:
I see that you actually have both further down, but I would be careful to let users know that the revision you pass to `west init` via `--mr` does not in any way "lock" your local copies to a specific tag/release. A bit of clarification would help there.

Reply:
I've rewritten the paragraph. I disagree though about what users will "typically" do. Those who actually read and follow the documentation will probably stick to one release as long as they can.

Reply:
Very well, thank you